docker/settings.py

```diff
@@ -92,3 +92,9 @@
     'oauth_secret': os.getenv('SPOTSEEKER_WEB_OAUTH_SECRET'),
     'oauth_user': os.getenv('SPOTSEEKER_WEB_OAUTH_USER'),
 }
+
+SPOTSEEKER_TECHLOAN_URL = os.getenv(
+    'SPOTSEEKER_TECHLOAN_URL',
+    ('https://asa-django-01.asa.uw.edu/techloan/api/v2/type/'
```

Comment:
Let's remove this as the default - preferably we should set it to None and raise ImproperlyConfigured if the code that requires it will error with a None value.